gef-ui/server/utils/helpers.js

```diff
@@ -413,11 +413,15 @@ const areUnissuedFacilitiesPresent = (application) => {
    from date of submission
 */
 const facilityIssueDeadline = (submissionDate) => {
-  // converts to timestamp from epoch - '+' to convert from str to int
-  const date = new Date(+submissionDate);
-  const deadlineDate = add(new Date(date), { months: 3 });
+  if (submissionDate) {
+    // converts to timestamp from epoch - '+' to convert from str to int
+    const date = new Date(+submissionDate);
```

Comment:
Instead of using unary plus, shall we use native `Number()` or `parseInt()`?

Reply:
@Zainzzkk ? I did not add this, just moved the comment

Reply:
@Zainzzkk happy for me to do this in my current PR?

Reply:
Yeah that is fine, easier for others to understand whats happening if parseInt actually